src/main/java/uk/ac/sanger/sprint/SPrintApplication.java

```diff
@@ -1,13 +1,33 @@
 package uk.ac.sanger.sprint;
 
+import org.apache.catalina.connector.Connector;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory;
+import org.springframework.boot.web.servlet.server.ServletWebServerFactory;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class SPrintApplication {
 
+    @Value("${http.port:0}")
+    private int httpPort;
+
     public static void main(String[] args) {
         SpringApplication.run(SPrintApplication.class, args);
     }
 
+    // Configure additional connector to enable support for both http and https
```

Comment:
Is the default now HTTPS, so the extra connector has to be added to keep supporting HTTP?

Reply:
Yes exactly that.